### PR TITLE
Add docker image repository to node details

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java [hunk 1]
```diff
@@ -548,6 +548,15 @@ private void logEndpoints(Map<ZoneId, List<Endpoint>> zoneEndpoints, DualLogger
                                                                                                            : " has config generation " + service.currentGeneration() + ", wanted is " + node.wantedConfigGeneration())));
     }
 
+
+    private String wantedPlatform(Node node) {
+        return node.wantedDockerImage().repository() + ":" + node.wantedVersion();
+    }
+
+    private String currentPlatform(Node node) {
+        return node.wantedDockerImage().repository() + ":" + node.wantedVersion();
```
Comment (freva) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java:557]: wanted -> current

**Review by freva — COMMENTED** [review]
How about only showing repository if they are different, at least for current?
```
--- platform docker.ouroath.com:4443/vespa/hosted:7.210.10 <-- 7.209.9
--- platform docker.ouroath.com:4443/vespa/hosted:7.210.10 <-- docker.ouroath.com:4443/vespa/ci:7.209.9
```
vs
```
--- platform docker.ouroath.com:4443/vespa/hosted:7.210.10 <-- docker.ouroath.com:4443/vespa/hosted:7.209.9
--- platform docker.ouroath.com:4443/vespa/hosted:7.210.10 <-- docker.ouroath.com:4443/vespa/ci:7.209.9
```

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java (hunk 1 above)
Comment (hmusum) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java]: Fixed, and did your suggestion about only showing current if different from wanted

**Comment by hmusum** [comment]
@freva PTAL

**Review by freva — APPROVED** [review]
Great!

Sorry, I'm always not sure what to do when I just comment on a PR I wasn't originally assigned on :slightly_smiling_face:

**Comment by hmusum** [comment]
Appreciate your comments, it's my responsibility to ask when that happens, so no worries.

**Pull request merged**